tests/test_templates.py

```diff
@@ -74,3 +74,174 @@ def test_service_active_class_soo():
+            'guidance_operations_and_compliance': (
+                'http://operations-and'
+                '-compliance.com'),
+            'services_fab': 'http://export.com/fab',
+            'services_soo': 'http://export.com/soo',
+            'services_exopps': 'http://export.com/exopps',
+            'services_get_finance': 'http://export.com/get-finance',
+            'services_events': 'http://export.com/events',
+            'info_about': 'http://about.com',
+            'info_contact_us': 'http://contact.com',
+            'info_privacy_and_cookies': 'http://privacy-and-cookies.com',
+            'info_terms_and_conditions': 'http://terms-and-conditions.com',
+            'info_dit': 'http://dit.com',
+        }
+    }
+    html = render_to_string(template_name, context)
+    hf_urls = context['header_footer_urls']
+    assert hf_urls['great_home'] in html
+    assert hf_urls['great_export_home'] in html
+    assert hf_urls['new_to_exporting'] in html
+    assert hf_urls['occasional_exporter'] in html
+    assert hf_urls['regular_exporter'] in html
+    assert hf_urls['guidance_market_research'] in html
+    assert hf_urls['guidance_customer_insight'] in html
+    assert hf_urls['guidance_finance'] in html
+    assert hf_urls['guidance_business_planning'] in html
+    assert hf_urls['guidance_getting_paid'] in html
+    assert hf_urls['guidance_operations_and_compliance'] in html
+    assert hf_urls['services_fab'] in html
+    assert hf_urls['services_soo'] in html
+    assert hf_urls['services_exopps'] in html
+    assert hf_urls['services_get_finance'] in html
+    assert hf_urls['services_events'] in html
+    assert hf_urls['info_about'] in html
+    assert hf_urls['info_contact_us'] in html
+    assert hf_urls['info_privacy_and_cookies'] in html
+    assert hf_urls['info_terms_and_conditions'] in html
+    assert hf_urls['info_dit'] in html
+
+
+def test_urls_exist_in_old_footer():
+    template_name = 'directory_header_footer/footer_old.html'
+    context = {
+        'header_footer_urls': {
+            'great_export_home': 'http://export.com',
+            'new_to_exporting': 'http://export.com/new',
+            'occasional_exporter': 'http://export.com/occasional',
+            'regular_exporter': 'http://export.com/regular',
+            'services_fab': 'http://export.com/fab',
+            'services_soo': 'http://export.com/soo',
+            'services_exopps': 'http://export.com/exopps',
+            'services_get_finance': 'http://export.com/get-finance',
+            'services_events': 'http://export.com/events',
+            'info_about': 'http://about.com',
+            'info_contact_us': 'http://contact.com',
+            'info_privacy_and_cookies': 'http://privacy-and-cookies.com',
+            'info_terms_and_conditions': 'http://terms-and-conditions.com',
+            'info_dit': 'http://dit.com',
+        }
+    }
+    html = render_to_string(template_name, context)
+    hf_urls = context['header_footer_urls']
```

Comment:
please avoid abbreviations in variable names and instead optimize for quick comprehension by the reader - so instead of `hf_urls`, this could be `header_footer_urls`, `context_urls`, or even simply `urls`.

The reason to avoid the abbreviation is that it makes the reader infer the meaning. After a few seconds I realised it meant "header and footer urls", but let's reduce the amount of thinking the reader has to do :) Note this "code comprehension tax" on the reader is especially compounded if multiple abbreviations are used.

Reply:
mainly did this to avoid more line length errors but checking now 'header_footer_urls' juuust about fits. will update